src/components/page-tabs/page-tabs.tsx

```diff
@@ -17,6 +18,7 @@ export default function PageTabs({
         setSelectedTab(activeKey);
       }}
       overrides={overrides.tabs}
+      endEnhancer={endEnhancer}
```

Comment:
Can we call this prop something else? Because individual tabs also have end enhancers, and it can get confusing